exchanges/bitfinex/bitfinex_websocket.go

```diff
@@ -926,7 +928,9 @@ func (b *Bitfinex) handleWSNotification(d []interface{}, respRaw []byte) error {
 		case strings.Contains(channelName, wsOrderUpdateRequest),
 			strings.Contains(channelName, wsOrderCancelRequest):
 			if data[0] != nil {
-				if id, ok := data[0].(float64); ok && id > 0 {
+				if id, ok := data[0].(float64); !ok {
+					return common.GetTypeAssertError("float64", data[0])
```

Comment:
```suggestion
					return common.GetTypeAssertError("float64", data[0], "wsOrderUpdateRequest/wsOrderCancelRequest id")
```

Reply:
Think better to use chanName for both of these, since it's non-internal and consumers can look at BFX api docs.
Now I'm looking at it, these strings.Contains all seem superfluous; equality checks should work.
Not going to fix that now.